### ISSUE TITLE
Include a font with TT debugging glyphs? 

### ISSUE THREAD

**Issue by aaronbell** [googlefonts/how-to-hint-variable-fonts#23]
It could be useful to include a font here that contains the [TrueType mode debugging glyphs](https://github.com/unicode-org/text-rendering-tests/blob/master/fonts/Selawik-README.md#debugging-glyphs) as it can be useful for better understanding what's going on. For example, even putting GDI rendering environments aside, there are some DWrite environments that don't support y-direction smoothing, which can cause a great deal of confusion as to why your perfectly-hinted font is not rendering correctly. 

While this sort of thing isn't necessarily a "all the time" sort of tool, it can certainly be handy.

**Comment by mikedug** [comment]
Thanks Aaron. Was away / offline last week. Glad you found the document useful!. 

I am curious to know which DirectWrite environments do not support y-direction smoothing? and or do you have an example of where this caused confusion?

**Comment by aaronbell** [comment]
@mikedug I believe the older (2015 / 2019) version of Visual Studio (not Code) runs DWrite without subpixel rendering or y-direction antialiasing (ttmode 006).

Admittedly such is an unusual case, but there was no way to debug the problem without the special glyphs to test.

**Comment by chrissimpkins** [comment]
Would the addition of a link to the Selawik debugging project suffice Aaron?  Or do you feel that we need a brand new testing typeface?

**Comment by aaronbell** [comment]
Yes. It includes more than strictly necessary for our purposes here but it does what needs doing :)
